spec/integration/rails/finders_spec.rb

```diff
@@ -110,6 +110,14 @@ def resource
                                created_at_date: 1.day.from_now.to_date
       end
 
+      context "when multiple operators" do
```

Comment:
This is fine to keep here, but ideally would be a PORO spec in `spec/filtering_spec.rb`